docs/connector-development/airbyte101.md

```diff
@@ -0,0 +1,6 @@
+# Airbyte 101 for Connector Development
```

Comment:
What is the purpose of that page?

Reply:
@sherifnada and I decided that we need a good home for concepts that are useful to understand when developing connectors. We needed a way to link to the `Beginner's guide to Catalog` doc and this is the way that we came up with.